themes/default/content/blog/pulumi-release-notes-87/index.md

```diff
@@ -0,0 +1,191 @@
+
+Two new community packages were added to the Registry: 
+
+- [AzAPI](https://www.pulumi.com/registry/packages/azapi/) - shoutout to [Engin Diri](https://github.com/dirien)
+- [StrongDM](https://www.pulumi.com/registry/packages/sdm/) - shoutout to [Piers Karsenbarg](https://github.com/pierskarsenbarg)
+
+## Pulumi Cloud & Pulumi.com
+
+### Resource Search Improvements
+
+![Screenshot of Resource Search improvements](data.png)
+
+**More Advanced Filtering**
+
+Based on customer feedback we have extended the number of advanced filtering options (Stack, Project, Package and Type) to show the top 10 results by resource count instead of top 3. We know our customers are getting a ton of value from these aggregates to inform and refine their searches and expect the 7 additional values to improve this further.
+
+**Pulumi Teams support**
+
+Customers who use Pulumi Teams for role based access control on Pulumi Cloud can now filter resources by team access. We have added a new aggregate on the Resource Search page for Teams, allowing you to see at a glance the teams in your organization with the most resources. You can query which teams have access to what resources, track resource usage by team by exporting your resource data and see in the search results table which teams have access to that resource.
+
+### Events Dashboard card
+
+![Events dashboard card](events.png)
+
+We have added a new card to the organizations dashboard which shows upcoming Pulumi events and Pulumi workshops to give users a central place to see events of interest.
+
+### Stacks Page Icons
+
+![Stacks page screenshot](stacks_page.png)
+
+We have added icons on the Stacks page to enable you to see at a glance information about your Pulumi Stacks, such as if Policies are being applied, if Pulumi Deployments is configured for that stack and the resource count. In addition, there is now a quick action menu on each stack to jump to configuring Pulumi Deployments and adding it to a Policy.
+
+### Deployment Actions
+
+You can now view the Pulumi Deployments Click-to-Deploy menu on all stack pages, even if not configured to Pulumi Deployments. This allows users to quickly navigate to Deployments Settings to connect the stack.
+
+![Stack page screenshot](deploy_actions.png)
+
+### SAML Admin
+
+For customers on the Enterprise or Business Critical Editions of Pulumi Cloud there is a new role for a SAML admin in your organization. SAML Admins have a email and password for accessing the organization when SAML is the only permitting authentication method. We are slowly rolling this out across our customer base, if it is not yet in your organization you should expect it in the coming weeks.
```

Comment:
```suggestion
For customers on the Enterprise or Business Critical Editions of Pulumi Cloud there is a new role for a SAML admin in your organization. SAML Admins can add another identity to authenticate when SAML is the only permitting authentication method.  This could be an email and password or another authentication method, like GitHub. We are slowly rolling this out across our customer base, if it is not yet in your organization you should expect it in the coming weeks.
```